### PR TITLE
Visual polishing

### PR DESCRIPTION
**Pull request by djrm**
fixed several ugly margins, improved a lot of colors, added some icons

### CODE REVIEW

**Comment by akien-mga** [comment]
Needs screenshot :)

File: editor/editor_themes.cpp [hunk 1]
```diff
@@ -480,6 +480,7 @@ Ref<Theme> create_editor_theme() {
 
 	//RichTextLabel
 	theme->set_stylebox("focus", "RichTextLabel", make_empty_stylebox());
+	theme->set_stylebox("focus", "RichTextLabel", make_empty_stylebox());
```
Comment (toger5) [inline editor/editor_themes.cpp]: Is this line duplicated?

File: editor/property_editor.cpp [hunk 2]
```diff
@@ -434,14 +433,14 @@ bool CustomPropertyEditor::edit(Object *p_owner, const String &p_name, Variant::
...
-				type_button->set_anchor_and_margin(MARGIN_TOP, ANCHOR_END, 25);
-				type_button->set_anchor_and_margin(MARGIN_BOTTOM, ANCHOR_END, 7);
+				easing_draw->set_anchor_and_margin(MARGIN_LEFT, ANCHOR_BEGIN, 5 * EDSCALE);
+				easing_draw->set_anchor_and_margin(MARGIN_RIGHT, ANCHOR_END, 5 * EDSCALE);
+				easing_draw->set_anchor_and_margin(MARGIN_TOP, ANCHOR_BEGIN, 5 * EDSCALE);
+				easing_draw->set_anchor_and_margin(MARGIN_BOTTOM, ANCHOR_END, 30 * EDSCALE);
+				type_button->set_anchor_and_margin(MARGIN_LEFT, ANCHOR_BEGIN, 3 * EDSCALE);
+				type_button->set_anchor_and_margin(MARGIN_RIGHT, ANCHOR_END, 3 * EDSCALE);
+				type_button->set_anchor_and_margin(MARGIN_TOP, ANCHOR_END, 25 * EDSCALE);
+				type_button->set_anchor_and_margin(MARGIN_BOTTOM, ANCHOR_END, 7 * EDSCALE);
```
Comment (toger5) [inline editor/property_editor.cpp]: Is this fuxing hidpi property editor inputs overlaping?

**Comment by toger5** [comment]
Really nice changes!

File: editor/property_editor.cpp (hunk 2 above)
Comment (djrm) [inline editor/property_editor.cpp]: @toger5 i dont know, but i've noticed that EDSCALE is missing in many custom sizes.

File: editor/editor_themes.cpp (hunk 1 above)
Comment (djrm) [inline editor/editor_themes.cpp]: oh yeah, i dont know what happened

**Comment by djrm** [comment]
@akien-mga changes are very minimal for screenshots, you will have to trust me ;)

**Pull request merged**